website/docs/introduction.md

```diff
@@ -10,61 +10,81 @@ This is a **very early and incomplete alpha release** of Docusaurus 2. We are ma
 **You should use this if**
 
 - :white_check_mark: You want to contribute to Docusaurus 2
-- :white_check_mark: You want to give feedback to make sure it suits your needs
+- :white_check_mark: You want to provide feedback to make sure it suits your needs
 - :white_check_mark: You are curious about what's next as a Docusaurus user
 
 **Do not use this if**
 
-- :x: You are expecting a full production-ready solution (<small>try [Docusaurus 1](https://docusaurus.io/) instead</small>)
-- :x: You are not willing to accept the breaking changes/things not working properly as we improve it during alpha period
+- :x: You need a full production-ready solution (<small>try [Docusaurus 1](https://docusaurus.io/) instead</small>)
+- :x: You prefer not to work with potential breaking changes and/or features not yet working properly as we improve it during alpha period
 
-## Introduction
+## The next Docusaurus is coming to town
 
 <img src="https://docusaurus.io/img/slash-introducing.svg" alt="Docusaurus"/>
 
-Docusaurus 1 used to be a documentation site generator. In Docusaurus 2, we rebuilt it from the ground up, allowing for more customizability but preserved the best parts of Docusaurus 1 - easy to get started. Docusaurus 2 is a **performant static site generator** can be used to create most content-driven websites (e.g. Documentation, Blogs, Product Landing and Marketing Pages, etc) extremely fast. Out of the box, we have support for documentation-style and blogging websites.
+Docusaurus 1 used to be a pure documentation site generator. In Docusaurus 2, we rebuilt it from the ground up, allowing for more customizability but preserved the best parts of Docusaurus 1 - easy to get started, versioned docs, and i18n.
```

Comment:
lets omit this part first, just “easy to get started”. versioning and i18n is not yet done

Reply:
also i like the previous text of can be created to build blog, docs out of the box